locales/en/app.json

```diff
@@ -70,7 +70,8 @@
     "livekit_sfu": "LiveKit SFU: {{url}}",
     "matrix_id": "Matrix ID: {{id}}",
     "show_connection_stats": "Show connection statistics",
-    "show_non_member_tiles": "Show tiles for non-member media"
+    "show_non_member_tiles": "Show tiles for non-member media",
+    "use_NewMembershipmanager": "Use the new implementation of the call MembershipManager"
```

Comment:
The name is inconsistent with other keys... but as it's a dev setting it's not important.